app/src/main/java/com/weatherxm/ui/devicesettings/DeviceSettingsViewModel.kt

```diff
@@ -58,6 +58,7 @@ class DeviceSettingsViewModel(
                             Analytics.ParamValue.CHANGE_STATION_NAME_RESULT_ID.paramValue,
                             success = 1L
                         )
+                        device.friendlyName = friendlyName
                         onEditNameChange.postValue(friendlyName)
```

Comment:
I don't know if is easy, It would be better to use the API response to update the `friendlyName`. In any case, it's minor

Reply:
The API response doesn't contain the `friendlyName`, it's an empty body actually. 😞